AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -27,14 +27,10 @@ def activity(h5,state = 'track'):
 def w_average(arr,axis=None, weights=None):
     return np.nansum(arr*weights,axis=axis)/np.nansum(weights,axis=axis)
 
-def reduce_compscan_inf(h5 ,channel_mask = None,chunks=16,return_raw=False,use_weights=False,compscan_index=None,debug=False):
+def reduce_compscan_inf(h5,rfi_static_flags=None,chunks=16,return_raw=False,use_weights=False,compscan_index=None,debug=False):
     """Break the band up into chunks"""
     chunk_size = chunks
-    rfi_static_flags = np.tile(False, h5.shape[0])
```

Comment:
That's because this code was never run... The script always has a channel_mask handy.

My preference is to remove excessive default values (like `chunk=16`...) from what is essentially an internal function of a script (is anything else in kat-land calling this?).

Reply:
your suggestion to "remove excessive default values" is beyond the scope of this PR "Bug fixes for RFI static mask".

**these specific proposed code changes appear to be accepted**

Reply:
Phew, I had to figure out what I meant first, since this was pre-lockdown 😂 

I meant that there is no need to provide default values for parameters if this function is only ever called with fully specified parameters in line 320. Default values make sense for user-facing routines, and less so for internal functions like this one (unless you are calling this somewhere else?).

I would just remove this line entirely and make `rfi_static_flags` a required parameter. That makes  both the problem go away and the code shorter.

If this were my PR, I would upgrade the "bug fixes for mask" to include "excessive use of defaults" as the origin of the bug and fix that too, but you are of course welcome to leave that for another day. :-D